.coveragerc

```diff
@@ -1,11 +0,0 @@
-[run]
```

Comment:
The coverage report looks wrong. 
- It takes into account `.tox/...packages/...` instead of `packages/...` for packages
- Seems like we need to tell it to only cover the modules being tested here. Might have to create 2 coverage/test commands (one used in normal PRs and one for release PRs where the coverage is only the set of tests being tested ini both scenarios.

Reply:
still usure about it